### PR TITLE
Added alpha and beta params (for AR and TAR in RNN Trainer) + some typo fixes 

### PR DESCRIPTION
**Pull request by tpietruszka**
Sometimes it can be useful to change the AR/TAR parameters - so I've added them as to LMHyperParams as rnn_alpha and rnn_beta. Or should they be called something else?

+ some minor fixes

### CODE REVIEW

**Comment by tpietruszka** [comment]
Some additional commits got added to the PR, including 
- the option to train a backwards LM
- updates/fixes to tests

Initially almost all tests were failing for various reasons, e.g. the data/test directory not existing. 

Bigger problems - test_text_data seemed to be written before batch_size got to be the first dimension of all tensors. I have fixed that, hope its okay.

Now, for me, all tests pass except one `test_text_train.py` - `test_bilm_lstm_can_be_trained` - this one trains, but the accuracy is too low somehow.

@PiotrCzapla could you please review?

**Comment by PiotrCzapla** [comment]
Thank you for the PR. I'm busy finishing the original idea of applying ulmfit to multiple languages. That is running tests on MLDoc tests and add all datasets collected by ppl on fastai forum.
We are freezing experiments with new architectures for a bit as It doesn't give promising enough results and we need to have the repo up to date with fastai that breaks bidirectional training.

So I will let this PR hanging for a bit until we have master up to date with fastai and we start testing bidirectional training again.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 fire>=0.1.3
 cupy>=5.0.0
 scikit-learn>=0.20
-sacremoses>=0.0.5
\ No newline at end of file
+sacremoses>=0.0.5
+sentencepiece
```
Comment (PiotrCzapla) [inline requirements.txt]: thx for that I'll add that in a bit.

File: README.md [hunk 2]
```diff
@@ -88,3 +88,13 @@ $ git push --set-upstream n-waves ulmfit_multilingual  # to automatically push u
...
+
+
+## Running tests
+
+To run the tests, the following data is necessary:
+
+- wikitext-2 (prepared by `./prepare_wiki-en.sh`, along with wikitext-103)
+- imdb (prepared by `./prepare_imdb.sh`)
+
+then simply run tests, e.g. `pytest .`
```
Comment (PiotrCzapla) [inline README.md]: make sense thx.

**Pull request merged**